db/models/Episode.js

```diff
@@ -0,0 +1,36 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const episodeSchema = new Schema({
+  _id: { type: String, required: true },
```

Comment:
There is no need to specify the _id field in the schema, since it automatically being attached and required for every document.

Reply:
changed it